src/renderer/actions/network.ts

```diff
@@ -40,3 +46,41 @@ export const saveSelectedNetworkAction = (network: string): ISaveSelectedNetwork
     type: NetworkActionTypes.SELECT_NETWORK,
     payload: network,
 });
+
+export const saveBeaconNodeAction = (url: string) => {
+    return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
+        const signingKey = PrivateKey.fromBytes(
+            Buffer.from(getState().register.signingKey.replace("0x",""), "hex")
+        );
+        const beaconNode = new BeaconNode(url);
+        const validatorAddress = signingKey.toPublicKey().toHexString();
+        await database.beaconNodes.set(
+            validatorAddress,
+            beaconNode,
+        );
+    };
+};
+
+export const loadedBeaconNode = (node: BeaconNode | null, validatorAddress: string): ILoadBeaconNodeAction => ({
+    type: NetworkActionTypes.LOAD_BEACON_NODE,
```

Comment:
Add plural everywhere. Like `load_beacon_nodes`

Reply:
I would also suggest to name this loadValidatorBeaconNodes since loadBeaconNodes will be used to fetch all available beacon nodes